DOC-179 & DOC-175

AnneLaure1307 opened the pull request:
Impacted pages:

- Release notes 5.5
- Release notes 5.4.8

ElisaPenot commented on docs/thehive/release-notes/release-notes-5.5.md:
```diff
@@ -1 +1,94 @@
+!!! info "Public API v0 deprecation"
+    Version 5.5 is the last minor release to support API v0 routes, which will be deactivated by default in version 5.6. Note that API v0 has been deprecated since version 5.4.0.
+
+## 5.5.0 - April 3, 2025
```
date à modifier pour le 22/04

AnneLaure1307 commented:
@ElisaPenot I've updated the release date. I've also added the documentation updates in a new section as we discussed yesterday. I'll let you check if everything looks good to you. I didn't include the creation/updates of pages related to the new 5.5 features because the links are already included in the New features section.
